package.json

```
@@ -5,7 +5,7 @@
     "postinstall": "lerna bootstrap --hoist",
     "build": "lerna run build",
     "lint": "lerna run lint",
```

As we discussed yesterday, we could add `--parallel` here too.

```suggestion
    "lint": "lerna run lint --parallel",
```